app/lib/io/zipUtil.js

```diff
@@ -12,7 +12,26 @@ define([
     var NotImplementedError = errors.NotImplemented;
 
     var unpack = function(async, zipData, io, targetPath){
-        throw new NotImplementedError('ZIP unpack method is not yet implemented');
+        if (async)
+            throw new NotImplementedError('Asynchronous ZIP unpack method is not yet implemented');
+
+        var zip = new JSZip(zipData);
+        var files = zip.files;
+        console.log("unpacking files: ", files);
+        var i;
+
+        for (i in files){
```

Comment:
It is a dictionary (an object) where the keys are filenames and the values are file objects

for (filename in files)
    file = files[filename];